apps/balancer-tools/src/app/poolsimulator/(components)/Menu.tsx

```diff
@@ -232,6 +231,102 @@ function SearchPoolFormWithDataForm({
   );
 }
 
+export function AnalysisMenu() {
```

Comment:
Why move this component to here? 😅

Reply:
Since the `IndexMenu` was here I wanted to let both here 😅